.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
```

Comment:
Rename this to Publish Storybook, same with the filename

Reply:
Name changed